bedrock/settings/static_media.py

```diff
@@ -1171,6 +1180,20 @@
         ),
         'output_filename': 'js/firefox_new-bundle.js',
     },
+    'firefox_new2': {
+        'source_filenames': (
+            'js/libs/jquery-1.11.3.min.js',
+            'js/libs/spin.min.js',
+            'js/base/global.js',
+            'js/base/global-init.js',
+            'js/base/search-params.js',
+            'js/newsletter/form.js',
+            'js/base/mozilla-input-placeholder.js',
```

Comment:
Can we safely omit this file? I think only up-to-date Firefox users see the newsletter, in which case this polyfill shouldn't be needed.